### ISSUE TITLE
Pylint cannot recognize members of gmpy2

### ISSUE THREAD

**Issue by FantasqueX** [PyCQA/pylint#4436]
Hi there! I use pylint and gmpy2 on Kali Linux. But It seems that pylint cannot recognize members of gmpy2.

### Steps to reproduce

Given a file `tst.py`:
```python
# (Disable message unrelated to the bug)
# pylint: disable=missing-docstring
import gmpy2
print(gmpy2.is_even(6))
```

### Current behavior

Result of `pylint a.py`:
```
************* Module tst
tst.py:2:6: E1101: Module 'gmpy2' has no 'is_even' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -20.00/10, +5.00)
```

### Expected behavior

I can run this simple script correctly, so I think pylint shouldn't complain about this script.

### pylint --version output

Result of `pylint --version` output:

```
pylint 2.7.2
astroid 2.5.1
Python 3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110]
```

**Comment by Pierre-Sassoulas** [comment]
Hi, did you try adding ``gmpy2`` to the ``extension-pkg-allow-list`` ?

**Comment by FantasqueX** [comment]
> Hi, did you try adding `gmpy2` to the `extension-pkg-allow-list` ?

Sorry for the inconvenience caused for me. I haven't read the doc about  `extension-pkg-allow-list` before.

After reading the doc, I fixed the problem. Thanks again for your patience! I'll close the issue.

**Comment by Pierre-Sassoulas** [comment]
Actually the problem is with the pylint doc or with the no-member message. I opened https://github.com/PyCQA/pylint/issues/4441

**Comment by FantasqueX** [comment]
@Pierre-Sassoulas Before I opened this issue, I searched the Internet for the answer. But It didn't help. So I created an [answer ](https://stackoverflow.com/questions/67367733/pylint-cannot-recognize-gmpy2-members/67412059#67412059)on stackoverflow to help others.
I also think it is a problem. Looking forward to the enhancement!

**Comment by hippo91** [comment]
@FantasqueX thanks for your stackoverflow answer.
